src/core/i18n/de/translation.json

```diff
@@ -17,14 +25,14 @@
     "Samstag",
     "Sonntag"
   ],
-  "typical_day": "Ein wichtiger Teil dieses Projektes ist es, herauszufinden, wie Kinder ihre Zeit über die Woche hinweg verbringen. War gestern ein typischer {{day}} für Sie und Ihr Kind? Ein untypischer Tag würde eine Schulschließung, Urlaub oder krank Zuhause sein beinhalten.",
+  "typical_day": "Ein wichtiger Teil dieses Projektes ist es, herauszufinden, wie Kinder ihre Zeit über die Woche hinweg verbringen. War {{activityDay}} ein typischer {{day}} für Sie und Ihr Kind? Ein untypischer Tag würde eine Schulschließung, Urlaub oder krank Zuhause sein beinhalten.",
```

Comment:
resulting values for typical_day and typical_day_choices are correct (lines 28-32)